lib/moon/helpers/merge_class.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Moon.Helpers.MergeClass do
```

Comment:
https://github.com/coingaming/moon/issues/474

What is the reason we are not using Twix? 

Afaik there are classnames, that are not "bluntly naming grouped". For example display classes, etc:

https://github.com/bratsche/twix/blob/main/lib/twix/default_config.ex#L31